app/src/main/res/layout/context_menu.xml

```diff
@@ -12,16 +11,20 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_margin="0dp"
-        android:padding="0dp"
-        android:background="@android:color/transparent" />
+        android:background="@android:color/transparent"
+        android:paddingTop="0dp"
+        android:paddingStart="8dp"
+        android:paddingEnd="8dp"
+        android:paddingBottom="16dp" />
 
     <TextView
         android:id="@+id/warning"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:background="#ff262626"
+        android:paddingHorizontal="24dp"
+        android:paddingVertical="16dp"
```

Comment:
`paddingHorizontal/Vertical` are API 26+ and we support 21+ – can you change this to be start/end/top/bottom?